### DISCUSSION TITLE
Is it possible to omit all code from the PDF produced by quarto?

### DISCUSSION THREAD

**Discussion by phinate** [quarto-dev/quarto-cli#2832] [Q&A] [answered]
As in the title! I find the code listings are not needed for my application.

**Comment by cscheid** [comment, marked as answer]
https://quarto.org/docs/computations/execution-options.html

`echo: false`

**Reply by phinate** [reply]
Perfect, thanks! Don't know why I couldn't find this in the docs, I was too focused on looking at things labelled "code".

**Reply by phinate** [reply]
As a follow up: is this configurable per-output type? i.e. can I include the code in HTML but not pdf?

**Reply by cscheid** [reply]
```yaml
format:
  html:
    echo: false
  pdf: default
```

etc.
